src/styles/main.scss

```diff
@@ -0,0 +1,63 @@
+$size-watch: 300px;
+$size-center: 15px;
+
+@mixin square($size) {
+  width: $size;
+  height: $size;
+}
+
+.stopwatch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  @include square($size-watch);
+  border: 5px solid black;
+  border-radius: 50%;
+  margin: 50px;
```

Comment:
Check 7th point in the [checklist](https://github.com/mate-academy/layout_stop-watch/blob/master/checklist.md)